fuseops/ops.go

```diff
@@ -756,3 +756,66 @@ type ReadSymlinkOp struct {
 	// Set by the file system: the target of the symlink.
 	Target string
 }
+
+////////////////////////////////////////////////////////////////////////
+// eXtended attributes
+////////////////////////////////////////////////////////////////////////
+
+// Remove an extended attribute
```

Comment:
Please add comments that give some context about what this is and when it's sent. See `OpenFileOp` for example.